interfaces/builtin/libvirt.go

```diff
@@ -30,6 +30,7 @@ const libvirtBaseDeclarationSlots = `
 `
 
 const libvirtConnectedPlugAppArmor = `
+/run/libvirt/libvirt-sock-ro rw,
```

Comment:
from the manual:

> /var/run/libvirt/libvirt-sock-ro - the secondary socket for accessing libvirt APIs, with limited read-only privileges. A connection to this socket gives the ability to query the existence of objects and monitor some aspects of their operation. This is the socket that most management applications connect to when requesting read only mode. Typically this is what a monitoring app would use.

https://github.com/libvirt/libvirt/blob/50bca3d45b58a6dd51d3792cc856f4f49fe88bea/docs/daemons.rst?plain=1#L79